packages/x-components/src/x-modules/recommendations/wiring.ts

```diff
@@ -43,6 +50,9 @@ export const setRecommendationsExtraParams = wireCommit('setParams');
  * @internal
  */
 export const recommendationsWiring = createWiring({
+  RecommendationsChanged: {
+    setRecommendations
```

Comment:
Is this wire needed for something? AFAIK, `RecommendationsChanged` is emitted thanks to above change, when `$store.x.state.recommendations.recommendations` change, and then you set them again with the same value?